.travis.yml

```diff
@@ -89,39 +90,49 @@ install:
   ;;
   esac
 - cargo install --list
+
 script: |
   case "$FEATURE" in
 
   "fmt" )
-      cd exonum && cargo update && cd .. &&
-      cd sandbox && cargo update && cd .. &&
-      cd exonum && cargo audit && cd .. &&
+      cargo update &&
+      cd exonum &&
       cd exonum && cargo fmt -- --write-mode=diff && cd .. &&
-      cd sandbox && cargo fmt -- --write-mode=diff && cd ..
+      cd sandbox && cargo fmt -- --write-mode=diff && cd .. &&
+      cd testkit && cargo fmt -- --write-mode=diff && cd ..
   ;;
   "clippy-core" )
-      cd exonum && cargo clippy --verbose --features long_benchmarks  -- -D warnings
+      cd exonum && cargo clippy --verbose --features long_benchmarks -- -D warnings
   ;;
   "clippy-sandbox" )
       cd sandbox && cargo clippy --verbose -- -D warnings
   ;;
+  "clippy-testkit" )
```

Comment:
Seems like `testkit` jobs are not actually called...

Reply:
Fixed.